styles/contact.css

```diff
@@ -27,10 +31,67 @@
   left: 5%;
 }
 
-.contact_description{
-	margin-left: 15%;
+.intro{
+	text-align: center;
 }
 
-.contact_icons{
-	margin-left: 15%;
+.contact_container{
+	text-align: center;
+}
+
+
+.mail {
+	background: url("./../assets/mail_icon.svg") no-repeat;
+	background-size: contain;
+	width: 100px;
+	height: 100px;
+	position: relative;
+	margin: 0 auto;
+}
+
+.mail:hover {
+  background: url("./../assets/mail_icon_hover.svg") no-repeat;
+  background-size: contain;
+}
+
+.phone {
+	background: url("./../assets/phone_icon.svg") no-repeat;
+	background-size: contain;
+	width: 100px;
```

Comment:
Since the width, height, postion, and margin and similar across the four, why not make a generic contact_icon class that has those 4 things